### ISSUE TITLE
İ prefer animelist as list , season anime as grid . İf i change one , it also changes the other ones setting.

### ISSUE THREAD

**Issue by blackholeearth** [JICA98/DailyAL#28]
**Current behavior:**
İ prefer my watching/plantowatch/... list as List .
 seasonal anime Screen as Grid.

 İf i change one , it also changes the other one's setting.

**Expected behavior:**
My list display style vs  season anime display style
Should be separate.

**Comment by BedBug2479** [comment]
I agree add option to "change the view for current page only" option

Also add option to "change the view order for current page only" as well

**Comment by JICA98** [comment]
This bug is fixed in the latest release - https://github.com/JICA98/DailyAL/releases/tag/2025.8.1%2B100
